book/src/the-database/moo-value-types.md

```diff
@@ -2,24 +2,24 @@
 
 There are only a few kinds of values that MOO programs can manipulate:
 
-- integers (in a specific, large range)
-- real numbers (represented with floating-point numbers)
-- strings (of characters)
-- object numbers (of the permanent objects in the database)
-- object references (to the anonymous objects in the database)
-- bools
-- WAIFs
-- errors (arising during program execution)
-- lists (of all of the above, including lists)
-- maps (of all of the above, including lists and maps)
+- Integers (in a specific, large range)
+- Floats / Real numbers  (represented with floating-point numbers)
+- Strings (of characters)
+- Object numbers (references to the permanent objects in the database)
+- "Flyweights" - anonymous lightweight object _values_
+- Errors (arising during program execution)
+- Lists (of all of the above, including lists)
+- Maps (of all of the above, including lists and maps)
 
 ## Integer Type
 
-ToastStunt supports 64 bit integers, but it can also be configured to support 32 bit. In MOO programs, integers are written just as you see them here, an optional minus sign followed by a non-empty sequence of decimal digits. In particular, you may not put commas, periods, or spaces in the middle of large integers, as we sometimes do in English and other natural languages (e.g. 2,147,483,647).
+mooR integers are signed, 64-bit integers. 
 
-> Note: The values $maxint and $minint define in the database the maximum integers supported. These are set automatically with ToastCore. If you are migrating from LambdaMOO it is still a good idea to check that these numbers are being set properly.
+In MOO programs, integers are written just as you see them here, an optional minus sign followed by a non-empty sequence of decimal digits. In particular, you may not put commas, periods, or spaces in the middle of large integers, as we sometimes do in English and other natural languages (e.g. 2,147,483,647).
 
-## Real Number Type
+> Note: Many databases define the values $maxint and $minint. Core databases built for LambdaMOO or ToastStunt may not have values set which correspond to the correct (64-bit signed integer) maximum / minimum values for `mooR`.
```

Comment:
Maybe add something like:

In general, the integer range supported by `mooR` tends to be at least as large as the integer range supported by other servers (definitely true for LambdaMOO and ToastStunt), so this shouldn't lead to any errors.